src/components/App.tsx

```diff
@@ -77,9 +78,22 @@ export const App = observer(() => {
     }
   }, [handleDataContextChangeNotice]);
 
-  const handleComponentChangeNotice = useCallback((notification: ClientNotification) => {
+  const handleComponentChangeNotice = useCallback(async (notification: ClientNotification) => {
     if (notification.values.type === "graph") {
       sonificationStore.setGraphs();
+
+      // If this is an attribute change and the graph is sonifiable, automatically set it as selected.
+      if (notification.values.operation === "attributeChange") {
+        try {
+          const graphs = await getGraphDetails();
+          const graph = graphs.find((g: ICODAPGraph) => g.id === notification.values.id);
+          if (graph && isGraphSonifiable(graph)) {
+            sonificationStore.setSelectedGraphID(graph.id);
```

Comment:
Should this happen every a graph is updated, or only if a graph is updated and there is no currently selected graph?

Reply:
@lublagg In a Jira story comment, Dan said "I think it would be useful for the sonification menu to switch to the newest graph that is created **or modified** which is sonifiable. It seems the mostly likely thing would be for someone to create a graph and then want to sonify it, so making them switch the menu to that graph seems like extra work. It seems pretty easy to switch the menu selection to the graph with the latest notification that results in a sonifiable graph," (emphasis added). So, yeah, that is the idea. Whether that's odd or not, I'm not sure, but it is the intended functionality at this time.